qiskit_nature/runtime/vqe_client.py

```diff
@@ -74,6 +76,16 @@ def __init__(
             store_intermediate: Whether or not to store intermediate values of the optimization
                 steps. Per default False.
         """
+        warn_deprecated(
+            "0.6.0",
+            DeprecatedType.CLASS,
+            "VQEClient",
+            additional_msg=(
+                ". Instead you should update your code to use the Qiskit Runtime Primitives. For "
```

Comment:
Perhaps we should more say something like `use the new primitive based VQE with the Qiskit IBM Runtime Estimator primitive.` That would be a more direct "replacement" as such for places where one would have used VQEClient.